### ISSUE TITLE
Nina "Personal Success"

### ISSUE THREAD

**Issue by ninaparipovic** [dartmouth-cs98/21f-visuol#134]
# Nina Paripovic | Personal Goals
*what does success for this class mean to you?*

We have a fully functioning website that allows users to feel more confident about understanding their job offer after using the site. I want my friends to say that this tool is helpful. Success also means learning new skills and becoming more confident with React. 

## Challenges
*what are some challenges you foresee?*

Our team struggled a little bit with working on individual code and the timing of connecting the code together. At times, we had to wait on certain features to be added before others could continue building onto the site. This might be an issue we once again encounter this term. Another issue I see is finding a reasonable scope we can expand to. We are struggling a bit to predict exactly how long our features might take and whether we have time for more.
